src/data/schemas.js

```diff
@@ -0,0 +1,195 @@
+import { getEmailLink } from "../formatters/emails";
+import { getExpiryInfo } from "../formatters/expiry";
+import { getBugURL, getBugLinkTitle } from "../formatters/links";
+import { getSearchfoxLink } from "../formatters/searchfox";
+
+const REQUIRED_METRIC_PARAMS_DOCS =
+  "https://mozilla.github.io/glean/book/user/metric-parameters.html#required-metric-parameters";
+const OPTIONAL_METRIC_PARAMS_DOCS =
+  "https://mozilla.github.io/glean/book/user/metric-parameters.html#optional-metric-parameters";
+
+export const APPLICATION_DEFINITION_SCHEMA = [
+  {
+    title: "Source",
+    id: "url",
+    type: "link",
+    helpText: "Where the source code for this application lives.",
+  },
+  {
+    title: "Application ID",
+    id: "app_id",
+    type: "value",
+    helpText:
+      "The app's identifier exactly as it appears in the relevant app store listing (for relevant platforms) or in the app's Glean initialization call (for other platforms). In the case of the Google Play store, it may contain both dots and dashes. For applicable platforms, you should be able to construct an app store URL from this value.",
```

Comment:
```
In the case of the Google Play store, it may contain both dots and dashes.
```
Is that much detail needed here? Here the app id is read-only. They are defined in some other place, so I'm not sure how much that detail here really helps.

Reply:
Yeah, agreed, we can skip the google play store part. This was taken from the definition in probe-scraper